config/config.go

```diff
@@ -835,6 +835,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.consumable.endpoint", "https://e.serverbid.com/api/v2")
 	v.SetDefault("adapters.conversant.endpoint", "http://api.hb.ad.cpe.dotomi.com/cvx/server/hb/ortb/25")
 	v.SetDefault("adapters.cpmstar.endpoint", "https://server.cpmstar.com/openrtbbidrq.aspx")
+	v.SetDefault("adapters.criteo.endpoint", "https://bidder.criteo.com/cdb?profileId=230")
```

Comment:
I'm not sure if an endpoint is required for a default adapter. To set as disabled by default, you'll want to add:
```
v.SetDefault("adapters. criteo.disabled", true)
```

Reply:
Do you mind if I let the default endpoint and add the `v.SetDefault("adapters. criteo.disabled", true)` ?

Reply:
I don't mind.